packages/cli/src/modules/build/commands/package/start/startPackage.ts

```diff
@@ -17,9 +17,26 @@
 import { PackageRole } from '@backstage/cli-node';
 import { startBackend, startBackendPlugin } from './startBackend';
 import { startFrontend } from './startFrontend';
+import { statSync, readdirSync } from 'fs';
+import { resolve, join, parse } from 'path';
+
+export function resolveEntryPath(
+  entrypoint: string = 'dev',
+  targetDir: string,
+): string {
+  const { dir: entryDir, name: entryName } = parse(entrypoint);
+  const entryPath = join(entryDir, entryName);
+  const parentPath = resolve(targetDir, entryDir);
```

Comment:
Hm. Could all of the code above be using `paths.resolveTarget` instead [like for example here](https://github.com/backstage/backstage/blob/80f7b646a570b6537fabb6157fd8b37ea16b3081/packages/cli/src/modules/maintenance/commands/package/clean.ts#L21)? Then when you get the result of that, you can start doing stat and stuff on that target.

Reply:
Maybe it is not necessary after simplifying the logic, tell me what you think.